recommended-rules.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: contribute
+title: Recommended Rules
+---
+
+# Recommended Rules
+
+In the documentation, we provide a list of
+[recommended rules](https://docs.bazel.build/versions/master/rules.html).
+
+This is a set of high quality rules, which will provide a good experience to our
+users. We make a distinction between the supported rules, and the hundreds of
+rules you can find on the Internet.
+
+## Nomination
+
+If a ruleset meets the requirements below, a rule maintainer can nominate it
+to be part of the _recommended rules_ by filing a
+[GitHub issue](https://github.com/bazelbuild/bazel/).
+
+After a review by the [Bazel core team](https://bazel.build/governance.html), it
+will be recommended on the Bazel website.
+
+## Demotion
+
+If a ruleset stops meeting the requirements (e.g. the code is abandoned, not
```

Comment:
Some of the requirements below are vague (I agree with them, but it is difficult to objectively quantify; e.g. what level of test coverage is sufficient?).

I suggest establishing more process, how about smth like this:
- if there is a concern that a particular rule set is not meeting the requirements, a GitHub issue should be filed
- rule maintainers will be contacted and need to respond in X weeks (X=1?)
- Based on the outcome, Bazel core team might make a decision to demote the rule set

Reply:
Done